rest_api/config_apis/config-apis-index.adoc

```diff
@@ -13,7 +13,7 @@ toc::[]
 Description::
 +
 --
-APIServer holds configuration (like serving certificates, client CA and CORS domains) shared by all API servers in the system, among them especially kube-apiserver and openshift-apiserver. The canonical name of an instance is 'cluster'. 
+APIServer holds configuration (like serving certificates, client CA and CORS domains) shared by all API servers in the system, among them especially kube-apiserver and openshift-apiserver. The canonical name of an instance is 'cluster'.
```

Comment:
Not sure about removing the extra spacing in the rest_API docs. I think these changes will need to be reverted as well. I will get confirmation.

Reply:
+1 to not changing the API docs.